### DISCUSSION TITLE
zapret и точечная маршрутизация

### DISCUSSION THREAD

**Discussion by lopatin531** [bol-van/zapret#818] [Q&A]
Здравствуйте! прекрасно работает zapret на openwrt, но для полного идеала хотел бы иметь ещё и доступ к chatgpt или tidal, которые блокируются той стороной. Нашёл такую [статью](https://habr.com/ru/articles/767464/) на хабре. Сам я полный ноль в этих вопросах, поэтому использовал написанный автором скрипт для wireguard. Естественно, ничего не заработало, и zapret сломался. Насколько я понял, конфликт на уровне файервола. Можно ли как-то подружить? Пожалуйста, был бы признателен за чуть более подробный ответ, так так, увы, без технических навыков. Спасибо.

**Comment by Alex1305555** [comment]
Здравствуйте. Вам удалось решить проблему? Заработал Tidal?

**Reply by lopatin531** [reply]
Да, после покупки кинетика и установки запрета на него

**Reply by Alex1305555** [reply]
Через OpenWrt?

**Reply by lopatin531** [reply]
Нет, прошивка кинетика

**Comment by Alex1305555** [comment]
Понятно, а можете сделать скриншот с настройками? Если не сложно!?
